src/components/ebay-video/component.js

```diff
@@ -224,7 +224,10 @@ module.exports = {
         );
 
         // eslint-disable-next-line no-undef,new-cap
-        shaka.ui.Controls.registerElement('report', new Report.Factory());
+        shaka.ui.Controls.registerElement(
+            'report',
+            new Report.Factory(this.input.reportText || null)
```

Comment:
One more nit, sorry. I don't think there's a need to default to null?

You have the same thing here. I would just do `this.input.reportText` here.
https://github.com/eBay/ebayui-core/pull/1532/files#diff-ec1aeb41a76f173410fc83de66c8305a2e27f80a2053cb23a4b638495b445783R34

And same with it on the next one. If it's undefined or blank it should be fine I would assume right?